packages/opal-server/opal_server/server.py

```diff
@@ -297,6 +298,14 @@ async def shutdown_event():
 
     async def start(self):
         await load_scopes(self._scopes)
+        await self.sync_all_scopes()
+
+    async def sync_all_scopes(self):
+        from opal_server.worker import sync_scope
+
+        scopes = await self._scopes.all()
+        job = group([sync_scope.s(scope.scope_id) for scope in scopes])
+        job.apply_async().join()
```

Comment:
is this sync or async? will this block the reactor?

what if you are scheduling thousands of tasks on celery?
you don't want to block the entire worker on that.

Reply:
It's sync, I'm removing the `join()`, so it will not block, but we will have to handle the situation where clients access the server while it is downloading repos.